### DISCUSSION TITLE
`pip install pipenv` results in OSError: [WinError 2] The system cannot find the file specified: 'c:\python39\Scripts\virtualenv.exe'

### DISCUSSION THREAD

**Discussion by prativh24** [Atri-Labs/atrilabs-engine#342] [Help: Installation & Start] [answered]
Could not install packages due to an OSError: [WinError 2] The system cannot find the file specified: 'c:\\python39\\Scripts\\virtualenv.exe'

**Comment by darshitac11** [comment]
Which command had you run when you received this error?

**Reply by prativh24** [reply]
pip install pipenv

**Comment by cruxcode** [comment, marked as answer]
Add `c:\python39\Scripts\` in the PATH environment variable.

If that doesn’t work, follow this answer: https://stackoverflow.com/a/64585018
